tests/sklearn/test_adversarial_debiasing.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
```

Comment:
Could you add some simple comments to the test_*.py files? Just a couple of lines that discuss some non-obvious (if at all there is any) things should be fine.